base/setup/usetup/usetup.c

```diff
@@ -1874,7 +1874,8 @@ SelectPartitionPage(PINPUT_RECORD Ir)
 
 #define PARTITION_SIZE_INPUT_FIELD_LENGTH 9
 /* Restriction for MaxSize */
-#define PARTITION_MAXSIZE (pow(10, (PARTITION_SIZE_INPUT_FIELD_LENGTH - 1)) - 1)
+/* #define PARTITION_MAXSIZE(pow(10, (PARTITION_SIZE_INPUT_FIELD_LENGTH - 1)) - 1)*/
+#define PARTITION_MAXSIZE 99999999
```

Comment:
If partition maximum size calculation in AMD64 build of ROS doesn't work then it is probably because `pow` isn't actually implemented for the AMD64 architecture? So the legit fix would be to actually implement `pow` rather than rely on hacks like this?

Reply:
Sure, but calculating such a simple constant expression might not be necessary, and this value has only one usage.

Reply:
My point is, this piece of code never bothered anyone and the real fix should be implementing that math function. Since your patch is a mere workaround it must be treated and explicitly explained in a comment (including the aforementioned CORE-ID ticket) as such.

Reply:
I added a comment.
Should I rebase?

Reply:
You can just add a second commit. We'll merge everything when committing the PR.